pkg/podexecutor/staticpod.go

```diff
@@ -121,7 +127,9 @@ func (s *StaticPod) ControllerManager(apiReady <-chan struct{}, args []string) e
 		return staticpod.Run(s.Manifests, staticpod.Args{
 			Command: "kube-controller-manager",
 			Args: append(args,
-				"/usr/libexec/kubernetes/kubelet-plugins/volume/exec"),
+				"--flex-volume-plugin-dir=/usr/libexec/kubernetes/kubelet-plugins/volume/exec",
```

Comment:
this will break on systems (such as k3OS) with read-only `/usr` but I think this only emits a warning in the logs (over and over again).

Reply:
This changes is a bug fix to add the argument that was missing. This may need a larger conversation to address this concern.